jupiterone/questions/questions.yaml

```diff
@@ -1,4 +1,18 @@
 ---
-sourceId: managed:template
+sourceId: managed:terraform
 integrationDefinitionId: '${integration_definition_id}'
-questions: []
\ No newline at end of file
+questions: 
+- id: integration-question-terraform-cloud-users-with-no-multi-factor-authentication
+  title: Returns a list of Terraform Cloud users that do not use MFA in a Terraform team workspace or as an individual contributor?
```

Comment:
This title is a little hard to read 

Maybe:
“Are there any Terraform users that do not use MFA in a Terraform team workspace or as an individual contributor?”